src/library_glfw.js

```diff
@@ -57,6 +56,9 @@ var LibraryGLFW = {
       };
       this.buttons = 0;
       this.keys = new Array();
+      this.joyLast = 0; // GLFW_JOYSTICK_1
+      this.joys = {};
+      this.index2joy = {};
```

Comment:
Can you document here what is the purpose of these three variables. The name `index2joy` is a bit ad hoc, it would be good to have some details here.